docs/guide/samples-howto.md

```diff
@@ -91,6 +91,16 @@ In the second layout, the four motorized wheels are oriented in the same directi
 In the third layout, a simple [Ackermann steering geometry](https://en.wikipedia.org/wiki/Ackermann_steering_geometry) is shown.
 Note that more completed Ackermann steering geometry can be achieved using the [`AckermannVehicle` PROTO](../automobile/ackermannvehicle.md), and the [`car` library](../automobile/car-library.md).
 
+### [gears.wbt]({{ url.github_tree }}/projects/samples/howto/gears/worlds/gears.wbt)
+
+**Keywords**: gears, self-collision
+
+![gears.png](images/samples/gears.thumbnail.jpg) This example shows the transfer of power between two gears through
+physical interaction. The red gear is motorized whereas the blue one is passive, and when the `boundingObject` at the tip of the teeth
+collide the second wheel begins to turn. Using this sort of approach as general purpose transmission is not generally advised but
+does work with some tuning of the `dampingConstant` of the [HingeJoint](../reference/hingejoint.md), the ERP and CFM values in the
+[WorldInfo](../reference/worldinfo.md) node.
```

Comment:
In the documentation we write each sentence on one line and avoid to manually break sentences.
Could you please fix it?